### PR TITLE
#166842342 create reading stats model

### PR DESCRIPTION
**Pull request by Eazybee**
#### What does this PR do?
This PR  creates readingStats model

#### Description of Task to be completed?
- create readingStats model
- add associations to user and article
- write tests for readingStats model

#### How should this be manually tested?
- Run ```npm run test```

#### Any background context you want to provide?
- You need `node` and `npm` installed 
- Run ```npm install``` to install dependencies 
- You need to install an editorconfig extension in your vscode

#### What are the relevant pivotal tracker stories?
[#166842342](https://www.pivotaltracker.com/story/show/166842342)

### CODE REVIEW

File: server/tests/models/readingStats.spec.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+before(async () => {
+  try {
+    await sequelize.sync({ force: true });
+  } catch (error) {
+    console.log(error);
+  }
+});
+
+describe('Reading stats relationship', () => {
+  let newUser;
```
Comment (EmmanuelSage) [inline server/tests/models/readingStats.spec.js]: could the variable declaration be before the hook?

**Review by EmmanuelSage — APPROVED** [review]

**Comment by Sojisoyoye** [comment]
You might want to consider renaming the pull request to follow all lower case

**Review by Sojisoyoye — APPROVED** [review]
Looks good. consider changing the pull request name to 'create reading stats model'.

File: server/tests/models/readingStats.spec.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    }
+
+    const readArticleId = read[0].dataValues.ArticleId;
+    const readUserId = read[0].dataValues.UserId;
+
+    expect(newUser.id).to.be.equal(readUserId);
+    expect(newArticle.id).to.be.equal(readArticleId);
+  });
+
+  it('should not read article more than one', async () => {
```
Comment (benfluleck) [inline server/tests/models/readingStats.spec.js]: `should not read the article more than once`

**Pull request merged**